.travis/build.sh

```diff
@@ -18,70 +18,58 @@ exitIfError() {
   [ "$EXIT" != "0" ] && exit $EXIT
 }
 
+arch=$(uname -m)
+echo "arch: $arch"
```

Comment:
If you want to keep this, maybe it can be a bit nicer? `Architecture: `?

Reply:
Yes, let's have the printout to be sure that the actual architecture label is not something different then I'm expecting.
I can make it a bit nicer :)